packages/appeals-service-api/tests/unit/lib/notify.test.js

```diff
@@ -21,13 +21,14 @@ const mockSetReference = jest.fn().mockReturnThis();
 const mockSend = jest.fn();
 
 jest.doMock('@pins/common/src/lib/notify/notify-builder', () => {
-  return () => ({
+  const mock = {
```

Comment:
This could simply be returned directly, or implicitly returned by wrapping {} with ().

Reply:
It cant be, it think we discussed it at the time but it needs to be set as a var so it can be passed to the functions to mock the 'return this' functionality.